src/components/tasks/card/index.tsx

```diff
@@ -389,6 +399,49 @@ const Card: FC<Props> = ({
         </div>
     );
 
+    const handleProgressUpdate = () => {
+        if (
```

Comment:
formatting ?

Reply:
formatting is correct  but still, I run yarn lint-fix and yarn prettier: fix and also formatted doc of vs code